hoptimator-catalog/src/main/java/com/linkedin/hoptimator/catalog/ScriptImplementor.java

```diff
@@ -1,23 +1,32 @@
 package com.linkedin.hoptimator.catalog;
 
+import org.apache.calcite.rel.RelNode;
+import org.apache.calcite.rel.type.RelDataType;
+import org.apache.calcite.rel.type.RelDataTypeFactory;
+import org.apache.calcite.rel.type.RelDataTypeField;
+import org.apache.calcite.rel.type.RelDataTypeSystem;
+import org.apache.calcite.rel.rel2sql.RelToSqlConverter;
+import org.apache.calcite.rel.rel2sql.SqlImplementor;
 import org.apache.calcite.sql.SqlWriter;
 //import org.apache.calcite.sql.SqlWriterConfig;
```

Comment:
can this be removed?

Reply:
I'll add a comment. Leaving it in cuz it will be needed in next Calcite version.